.github/workflows/build-and-test.yml

```diff
@@ -112,7 +112,7 @@ jobs:
     - name: Validate Batect wrapper scripts
       uses: batect/batect-wrapper-validation-action@v0
     - name: Run functional tests
-      run: BATECT_QUIET_DOWNLOAD=true ./batect --enable-buildkit --config-var gradle_console=plain funTest -- jacocoFunTestReport -p analyzer
+      run: BATECT_QUIET_DOWNLOAD=true ./batect --enable-buildkit --config-var gradle_console=plain funTestAnalyzer
```

Comment:
Commit message nits:
- "Use a login shell as entrypoint" -> "Use a login shell as *the* entrypoint"
- "does not work anymore the pass additional arguments" -> "does not work anymore *to* the pass additional arguments"